python/ray/tune/execution/experiment_state.py

```diff
@@ -380,29 +381,54 @@ def sync_up(self, force: bool = False, wait: bool = False) -> bool:
         self._last_sync_time = now
         return True
 
+    def sync_down_experiment_state(self) -> None:
```

Comment:
Can we remove this and just read the exp json file directly here:

https://github.com/ray-project/ray/blob/ca1a546ddd769fac2f0dde9847e937dd89bed17e/python/ray/tune/execution/tune_controller.py#L528-L529